syft/rekor/utils.go

```diff
@@ -93,10 +94,11 @@ func parseEntry(entry *models.LogEntryAnon) (*models.IntotoV001Schema, error) {
 	return intotoV001, nil
 }
 
-// validateAttestation does some checks that the attestation contains the necessary fields to proceed
+// validateAttestation validates that the attestation contains the necessary fields to proceed, and
```

Comment:
nit: I think the function name changed here validateAttestation => validateAndGetFirsts